src/content/reference/react-dom/client/createRoot.md

```diff
@@ -45,7 +45,9 @@ An app fully built with React will usually only have one `createRoot` call for i
 
 * **optional** `options`: An object with options for this React root.
 
-  * **optional** `onRecoverableError`: Callback called when React automatically recovers from errors.
+  * <CanaryBadge title="This feature is only available in the Canary channel" /> **optional** `onCaughtError`: Callback called when React catches an error in an error boundary. Called with the `error` and an `errorInfo` object containing the `componentStack`.
+  * <CanaryBadge title="This feature is only available in the Canary channel" /> **optional** `onUncaughtError`: Callback called when an uncaught error is thrown. Called with the `error` and an `errorInfo` object containing the `componentStack`.
+  * **optional** `onRecoverableError`: Callback called when React automatically recovers from errors. Called with the `error` and an `errorInfo` object containing the `componentStack`. Some recoverable errors may include the original error cause as `error.cause`.
```

Comment:
Can we link to a section explaining what recovering from an error means in React?